CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,44 @@
 
 import Foundation
 
-print("Hello, World!")
+var myLottoNumbers: [Int] = []
+myLottoNumbers = [1, 2, 3, 4, 5, 6]
 
+checkWinningLottoNumbers(myNumbers: myLottoNumbers, winnigNumbers: makeLottoNumbers())
+
+func makeLottoNumbers() -> Set<Int> {
+    var lottoNumbers: Set<Int> = Set<Int>()
```

Comment:
type annotation을 사용하셨을 경우, 아래와 같이 인스턴스를 생성하기도 합니다. 👀
```suggestion
    var lottoNumbers: Set<Int> = .init()
```

Reply:
아하. 이런 방법도 있었네요.